Linux eslint issue

Error log attached. Issue is within naming of module which works fine on Mac but fails on linux.

Cannot find module 'eslint-config-konnektid/rules/base/bestpractices'
Referenced from: eslint-config-konnektid/rules/base
Referenced from: eslint-config-konnektid
Referenced from: konnektid/babel
Referenced from: /home/sergey/WebstormProjects/Konnektid/src/apps/konnektid-graph/.eslintrc
Error: Cannot find module 'eslint-config-konnektid/rules/base/bestpractices'
Referenced from: eslint-config-konnektid/rules/base
Referenced from: eslint-config-konnektid
Referenced from: konnektid/babel
Referenced from: /home/sergey/WebstormProjects/Konnektid/src/apps/konnektid-graph/.eslintrc
    at Object.ModuleResolver.resolve (/home/sergey/.npm-global/lib/node_modules/eslint/lib/util/module-resolver.js:75:19)
    at resolve (/home/sergey/.npm-global/lib/node_modules/eslint/lib/config/config-file.js:473:33)
    at load (/home/sergey/.npm-global/lib/node_modules/eslint/lib/config/config-file.js:490:24)
    at /home/sergey/.npm-global/lib/node_modules/eslint/lib/config/config-file.js:386:36
    at Array.reduceRight (native)
    at applyExtends (/home/sergey/.npm-global/lib/node_modules/eslint/lib/config/config-file.js:363:28)
    at load (/home/sergey/.npm-global/lib/node_modules/eslint/lib/config/config-file.js:525:22)
    at /home/sergey/.npm-global/lib/node_modules/eslint/lib/config/config-file.js:386:36
    at Array.reduceRight (native)
    at applyExtends (/home/sergey/.npm-global/lib/node_modules/eslint/lib/config/config-file.js:363:28)

--- a/rules/base/index.js
+++ b/rules/base/index.js
@@ -1,7 +1,7 @@
 module.exports = {
     extends: [
         "eslint-config-konnektid/rules/base/preventability",
-        "eslint-config-konnektid/rules/base/bestpractices",
+        "eslint-config-konnektid/rules/base/bestPractices",
         "eslint-config-konnektid/rules/base/variability",
         "eslint-config-konnektid/rules/base/brevity",
         "eslint-config-konnektid/rules/base/breathability",

